admin/check/check_paths_inc.php

```diff
@@ -74,18 +74,6 @@
 	$t_paths[$t_path_config_name] = $t_new_path;
 }
 
-# Trailing directory separator
-foreach( $t_paths as $t_path_config_name => $t_path ) {
-	check_print_test_row(
-		$t_path_config_name . ' configuration option has a trailing directory separator',
-		substr( $t_path['config_value'], -1, 1 ) == DIRECTORY_SEPARATOR,
-		array( false =>
-			'You must provide a trailing directory separator (' . DIRECTORY_SEPARATOR .
-			') to the end of \'' . htmlspecialchars( $t_path['config_value'] ) . '\'.'
-		)
-	);
-}
-
```

Comment:
I thought that the requirement for a path was no longer needed and... removed the whole check loop instead of the ‘graphviz_path’ option name. 😑 But now I think I should leave the check still, so that all paths are treated the same.